### ISSUE TITLE
Navigation to 'About' pages is confusing.

### ISSUE THREAD

**Issue by thomasfedb** [UDS-UWA/UDS-UWA.github.io#7]
The `menu -> popup -> 'more' -> menu page -> content` flow is confusing.

![ezgif com-crop](https://cloud.githubusercontent.com/assets/193831/6550046/c794511a-c659-11e4-8e96-fa6dd25fec52.gif)

**Comment by KaiStarkk** [comment]
Yeah I've already decided to abandon this theme. The navigation is gross (it doesn't work on lower resolution displays) and those buttons look godawful.

**Comment by KaiStarkk** [comment]
Grace suggested something with a more conventional top-bar dropdown navigation, I can look for something suitable or we can build one ourselves if it really comes down to it (would prefer to avoid this if possible).

**Comment by thomasfedb** [comment]
Either option suits me fine. I'd like something that lets us highlight the current production prominently.
